core/serv/README.md

```diff
@@ -13,7 +13,7 @@ The same binary acts as a command line interface for the service.
 | Data folder | `-d, --datadir <path>` | `YAGNA_DATADIR` | platform specific (see `--help`) | The folder in which the Daemon's SQL storage file is to be located | 
 | GSB URL | `-g, --gsb-url <url>` | `GSB_URL` | `tcp://127.0.0.1:7464` | Service Bus URL |
 | REST API URL | `-a, --api-url <url>` | `YAGNA_API_URL` | `http://127.0.0.1:7465` | Yagna REST API endpoints base URL |
-| Net Mk1 hub addr | N/A | `CENTRAL_NET_HOST` | `3.249.139.167:7464` | Centralized (Mk1 phase) Yagna network server address |
+| Net Mk1 hub addr | N/A | `CENTRAL_NET_HOST` | `yacn2.dev.golem.network:7464` | Centralized (Mk1 phase) Yagna network server address |
```

Comment:
this is not default value. Default is to get host and port from DNS

Reply:
`.env-template` file also needs updating with same description as here + Wiki https://github.com/golemfactory/yagna/wiki/DotEnv-Configuration

Reply:
I've removed the Wiki page. It makes no sense to have it tripled.

Reply:
updated